### ISSUE TITLE
Recommendations

### ISSUE THREAD

**Issue by willie1992** [MicrosoftDocs/OfficeDocs-Support#946]
[Enter feedback here]
Really need some pictures of the solution.  Hard to follow.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0d4c5396-5562-5f30-a7b6-c2fc045e553f
* Version Independent ID: 7efbc0ce-55fc-3e06-24df-a8f7a1f47afc
* Content: [All members' free/busy data of a group is displayed - Exchange](https://docs.microsoft.com/en-us/exchange/troubleshoot/outlook-issues/group-calendar-shows-all-user-free-busy-info)
* Content Source: [Exchange/ExchangeOnline/outlook-issues/group-calendar-shows-all-user-free-busy-info.md](https://github.com/MicrosoftDocs/OfficeDocs-Support/blob/public/Exchange/ExchangeOnline/outlook-issues/group-calendar-shows-all-user-free-busy-info.md)
* Product: **exchange-server-it-pro**
* GitHub Login: @simonxjx
* Microsoft Alias: **v-six**

**Comment by msbemba** [comment]
@willie1992 Thank you for your feedback. Sorry about the confusion. The Add to favorites is no longer available and so the confusion in navigation. I have added the correct steps to the document via PR#https://github.com/MicrosoftDocs/OfficeDocs-Support/pull/1391

1. Open your **Outlook** , expand **All Group Calendars**, and then select the modern group.
2. The issue will not occur when you view the calendar.

**Comment by nam31** [comment]
Closing via #1391 
Thank you
